src/View/Components/Card.php

```diff
@@ -2,67 +2,41 @@
 
 namespace WireUi\View\Components;
 
-use Illuminate\Support\Str;
+use Illuminate\Contracts\View\View;
+use Illuminate\Support\Arr;
 use Illuminate\View\Component;
 
 class Card extends Component
 {
     public function __construct(
+        public ?string $title = null,
         public ?string $padding = null,
         public ?string $shadow = null,
         public ?string $rounded = null,
         public ?string $color = null,
-        public ?string $title = null,
-        public ?string $action = null,
-        public ?string $header = null,
-        public ?string $footer = null,
-        public ?string $cardClasses = '',
-        public ?string $headerClasses = '',
-        public ?string $footerClasses = '',
     ) {
-        $padding ??= config('wireui.card.padding');
-        $shadow  ??= config('wireui.card.shadow');
-        $rounded ??= config('wireui.card.rounded');
-        $color   ??= config('wireui.card.color');
-
-        $this->padding       = $padding;
-        $this->shadow        = $shadow;
-        $this->rounded       = $rounded;
-        $this->color         = $color;
-        $this->cardClasses   = $this->setCardClasses($cardClasses);
-        $this->headerClasses = $this->setHeaderClasses($headerClasses);
-        $this->footerClasses = $this->setFooterClasses($footerClasses);
+        $this->padding = $padding ??= config('wireui.card.padding');
+        $this->shadow  = $shadow  ??= config('wireui.card.shadow');
+        $this->rounded = $rounded ??= config('wireui.card.rounded');
+        $this->color   = $color   ??= config('wireui.card.color');
```

Comment:
You can use this way

```suggestion
        $this->padding ??= config('wireui.card.padding');
        $this->shadow  ??= config('wireui.card.shadow');
        $this->rounded ??= config('wireui.card.rounded');
        $this->color   ??= config('wireui.card.color');
```